resources/views/layouts/partials/lang-switcher.blade.php

```diff
@@ -1,7 +1,8 @@
-<div class="text-center" style="border-bottom: 1px solid #e7e7e7; padding-bottom: 10px">
-    {{ trans('lang.lang') }} :
+<div class="text-center" style="border-bottom: 1px solid #e7e7e7; padding: 90px 10px 0px 0px">
+    {{-- {{ trans('lang.lang') }} : --}}
 
     @foreach (['en', 'id', 'de'] as $langKey)
+        {{-- @break --}}
```

Comment:
Remove this

Reply:
@nafiesl  If I remove this, the GitHub auto Travis build for this project fails. The failure is in a UI test. Specifically, `UserProfileTest->user_can_switch_lang_from_sidebar()` test fails

Reply:
Ok, i will see what i can do to fix this.